src/components/ProductCategoriesDialog.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-dialog>
+    <v-card>
+      <v-card-title>
+        {{ $t('ProductCategories.Title') }} <v-btn style="float:right;" variant="text" density="compact" icon="mdi-close" @click="close"></v-btn>
+      </v-card-title>
+
+      <v-divider></v-divider>
+
+      <v-card-text v-if="categories.length">
+        <v-chip v-for="category in categories" :key="category" label class="mr-2 mb-2">
+          {{ category }}
```

Comment:
I would use `category.name` here so that non-english speaker can use the information.

And eventually in the future have a small symbol next to untranslated names with a tooltip saying that category is not translated with a link to taxonomies.
Also a link to add categories for this product.

Reply:
the `categories` object looks like this : `["en:breakfasts", "en:spreads"]` . so the `category` is only a string

Reply:
Alright!